javascriptv3/example_code/dynamodb/scenarios/dynamodb_basics/libs/ddbDocClient.js

```diff
@@ -0,0 +1,43 @@
+/* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+ABOUT THIS NODE.JS EXAMPLE: This example works with the AWS SDK for JavaScript version 3 (v3),
+which is available at https://github.com/aws/aws-sdk-js-v3.
+
+Purpose:
+ddbDocClient.js is a helper function that creates an Amazon DynamoDB service document client.
+
+INPUTS:
+- REGION
+
+*/
+// snippet-start:[dynamodb.JavaScript.scenario.basics.createdocclientv3]
+
+// Create service client module using ES6 syntax.
+import { DynamoDBDocumentClient} from "@aws-sdk/lib-dynamodb";
+import {ddbClient} from "./ddbClient.js";
+// Set the AWS Region.
+const REGION = "REGION"; //e.g. "us-east-1"
```

Comment:
const REGION = "REGION"; //For example, "us-east-1".

Reply:
done